at_client/src/main/resources/config.yaml

```diff
@@ -1,4 +1,7 @@
-API_KEY: "477b-876u-bcez-c42z-6a3d" ##staging api key. please obtain a key to the production registrar api and replace it
-ROOT_DOMAIN: "root.atsign.wtf" ##staging root server. use "root.atsign.org" to use the production root domain
-ROOT_PORT: "64" 
-REGISTRAR_URL: "https://my.atsign.wtf/api/app/v2" ##staging registrar url. please use "https://my.atsign.com/api/app/v2" to use the production registrar api
\ No newline at end of file
+rootServer:
+  domain: "root.atsign.wtf" ##staging root server. use "root.atsign.org" to use the production root domain
+  port: "64"
+
+registrar:
+  url: "https://my.atsign.wtf/api/app/v2" ##staging registrar url. please use "https://my.atsign.com/api/app/v2" to use the production registrar api
+  apiKey: "477b-876u-bcez-c42z-6a3d" ##staging api key. please obtain a key to the production registrar api and replace it
```

Comment:
Please re-add the original config names, specifying in comments that they are there for backwards compatibility and that users should use, for example, rootServer:domain rather than ROOT_DOMAIN as ROOT_DOMAIN will be deprecated in future